### PR TITLE
Make highlighted search results legible

### PR DESCRIPTION
**Pull request by HatScripts**
Regardless of the chat bubble colour.

dark-messenger-search-result-highlights

### CODE REVIEW

File: fb-messenger-dark.css [hunk 1]
```diff
@@ -51,6 +51,12 @@ input::-webkit-input-placeholder {
 ._33p7 {
 	background-color: rgba(30, 30, 30, 0.9)	!important;
 }
+/* highlighted search results */
+.__in {
+	background-color: rgba(0, 0, 0, 0.3);
+	box-shadow: 2px 0 rgba(0, 0, 0, 0.3), -2px 0 rgba(0, 0, 0, 0.3);
+	padding: 1px 0 2px 0;
```
Comment (auscompgeek) [inline fb-messenger-dark.css:58]: Is this padding change really necessary? Clearly the original is what Facebook intends this to look like… /s

No, but seriously, I don't think this style should be fixing Facebook bugs, no matter how tiny they are.

File: fb-messenger-dark.css [hunk 2]
```diff
@@ -51,6 +51,12 @@ input::-webkit-input-placeholder {
 ._33p7 {
 	background-color: rgba(30, 30, 30, 0.9)	!important;
 }
+/* highlighted search results */
+.__in {
+	background-color: rgba(0, 0, 0, 0.3);
+	box-shadow: 2px 0 rgba(0, 0, 0, 0.3), -2px 0 rgba(0, 0, 0, 0.3);
```
Comment (auscompgeek) [inline fb-messenger-dark.css:57]: I'm pretty sure you'll want these to be `!important` for this to actually work consistently when installed as a userscript.

**Review by auscompgeek — CHANGES_REQUESTED** [review]
Looks good otherwise.

File: fb-messenger-dark.css (hunk 1 above)
Comment (HatScripts) [reply fb-messenger-dark.css:58]: It's necessary for me to maintain my sanity lol

messenger-bubble-padding-comparison

But seriously I don't consider that a bug—just an imperfection—and even if it were an actual bug, I highly doubt they'd ever bother fixing it. I [reported](https://github.com/auscompgeek/fb-messenger-dark/pull/18#issuecomment-298829333) the scrollbar in the input field bug to them 10 months ago, received no response and that bug still exists today.

File: fb-messenger-dark.css (hunk 2 above)
Comment (HatScripts) [reply fb-messenger-dark.css:57]: Ah yeah, you're probably right.

The thing is, I currently have these styles without `!important` in the following userscript and it still overrides the default styles. I recall [this having been a problem before](https://github.com/auscompgeek/fb-messenger-dark/pull/17#issuecomment-288868948). Do you know what causes it?

```js
// ==UserScript==
// @name        Custom Messenger Changes
// @namespace   https://github.com/HatScripts
// @version     DEV
// @author      HatScripts
// @match       http*://www.messenger.com/*
// @run-at      document-start
// @require     https://rawgit.com/HatScripts/UserscriptHelpers/master/applyCss.min.js
// ==/UserScript==

applyCss(`
._2swd._kmc {
	padding-bottom: 9px !important;
}

.__in {
	background-color: rgba(0, 0, 0, 0.3);
	box-shadow: 2px 0 rgba(0, 0, 0, 0.3), -2px 0 rgba(0, 0, 0, 0.3);
	padding: 1px 0 2px 0;
}
`);
```

Comment (auscompgeek) [reply fb-messenger-dark.css:57]: There's a potential race condition where messenger.com's JS finishes before userscripts might have a chance to run.

File: fb-messenger-dark.css (hunk 1 above)
Comment (auscompgeek) [reply fb-messenger-dark.css:58]: I would consider it to be a bug, as it goes into the space of the above line if the message is wrapped.

**Comment by auscompgeek** [comment]
Thanks for the contribution @HatScripts. I decided to cherry-pick your patch and address my above comments.

**Pull request closed without merging**